### ISSUE TITLE
AttributeError: 'Config' object has no attribute 'isolated_build_env'

### ISSUE THREAD

**Issue by adehad** [ymyzk/tox-gh-actions#96]
**Describe the bug**

The plugin seems to be having trouble after the latest release? 

**To Reproduce**
* GitHub Actions configuration: https://github.com/pycontribs/jira/blob/master/.github/workflows/jira_server_ci.yml#L53
* tox and tox-gh-actions configuration: https://github.com/pycontribs/jira/blob/master/tox.ini
* Link to the GitHub Actions job if a repository is public: https://github.com/pycontribs/jira/pull/1154/checks?check_run_id=3681261879#step:8:39

**Expected behavior**
That the plugin does not raise an exception when running on GitHub Actions

**Additional context**

<details>
  <summary>Dependencies Installed</summary>
  
```
 backports.entry-points-selectable-1.1.0 distlib-0.3.3 filelock-3.0.12 packaging-21.0 platformdirs-2.3.0 pluggy-1.0.0 py-1.10.0 pyparsing-2.4.7 six-1.16.0 toml-0.10.2 tox-3.24.4 tox-gh-actions-2.8.0 virtualenv-20.8.0
```

</details>

<details>
  <summary>Stack Trace</summary>
  
```
 Traceback (most recent call last):
.tox create: /home/runner/work/jira/jira/.tox/.tox
  File "/opt/hostedtoolcache/Python/3.8.12/x64/bin/tox", line 8, in <module>
    sys.exit(cmdline())
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/tox/session/__init__.py", line 44, in cmdline
    main(args)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/tox/session/__init__.py", line 69, in main
    exit_code = session.runcommand()
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/tox/session/__init__.py", line 187, in runcommand
    return provision_tox(provision_tox_venv, self.config.args)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/tox/session/commands/provision.py", line 10, in provision_tox
    ensure_meta_env_up_to_date(provision_venv)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/tox/session/commands/provision.py", line 24, in ensure_meta_env_up_to_date
    if provision_venv.setupenv():
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/tox/venv.py", line 629, in setupenv
    status = self.update(action=action)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/tox/venv.py", line 273, in update
    self.hook.tox_testenv_create(action=action, venv=self)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/pluggy/_hooks.py", line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/pluggy/_manager.py", line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/pluggy/_callers.py", line 60, in _multicall
    return outcome.get_result()
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/pluggy/_result.py", line 60, in get_result
    raise ex[1].with_traceback(ex[2])
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/pluggy/_callers.py", line 39, in _multicall
    res = hook_impl.function(*args)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/tox_gh_actions/plugin.py", line 66, in tox_testenv_create
    start_grouping_if_necessary(venv)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/tox_gh_actions/plugin.py", line 105, in start_grouping_if_necessary
    if envname == envconfig.config.isolated_build_env:
AttributeError: 'Config' object has no attribute 'isolated_build_env'
Error: Process completed with exit code 1.
```
</details>

**Comment by ymyzk** [comment]
Thanks for reporting an issue. I'm having a look now.

**Comment by ymyzk** [comment]
Preparing a fix in #97. It seems the trigger condition is when one or more `requires` is set in `tox.ini`.

```ini
[tox]
minversion = 3.8.0
requires =                                                                                  
     tox-extra 
     tox-pyenv                   
```

**Comment by ymyzk** [comment]
Just published v2.8.1 which fixes the issue. Thanks again for the detailed bug report! I'll have a deeper look later to understand the problem better and make the plugin more robust.

**Comment by adehad** [comment]
Awesome, thanks, I can confirm that the fix is working for us !
